sim/paladin/talents_retribution.go

```diff
@@ -46,6 +47,16 @@ func (paladin *Paladin) applyRuleOfLaw() {
 	})
 }
 
+func (paladin *Paladin) applyPursuitOfJustice() {
+	if paladin.Talents.PursuitOfJustice == 0 {
+		return
+	}
+
+	if paladin.PseudoStats.MovementSpeedMultiplier < 1.15 {
+		paladin.PseudoStats.MovementSpeedMultiplier = 1.15
```

Comment:
This should also be done with `core.ExclusiveEffect` to properly handle the effects it doesn't stack with.